src/Npgsql.OpenTelemetry/README.md

```diff
@@ -20,3 +20,42 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
 Once this is done, you should start seeing Npgsql trace data appearing in your application's console. At this point, you can look into exporting your trace data to a more useful destination: systems such as [Zipkin](https://zipkin.io/) or [Jaeger](https://www.jaegertracing.io/) can efficiently collect and store your data, and provide user interfaces for querying and exploring it.
 
 For more information, [visit the diagnostics documentation page](https://www.npgsql.org/doc/diagnostics.html).
+
+### Enrich
+
+This option allows one to enrich the activity with additional information from the `NpgsqlCommand`, or on any exception.
+The `Enrich` action is called only when `activity.IsAllDataRequested` is `true`.
+It contains the activity itself (which can be enriched), the name of the event, and either the `NpgsqlCommand` or an exception, depending on the event name:
+
+For event name "OnStartActivity", the actual object will be `NpgsqlCommand`.
+
+For event name "OnStopActivity", the actual object will be `NpgsqlCommand`.
+
+For event name "OnException", the actual object will be `Exception`.
+
+Example:
+
+```csharp
+using System;
+using Npgsql;
+using OpenTelemetry;
+
+var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .AddNpgsql(options => options.Enrich
+        = (activity, eventName, rawObject) =>
+        {
+            switch (eventName, rawObject)
+            {
+                case ("OnStartActivity", NpgsqlCommand command):
+                    activity.SetTag("command.type", command.CommandType);
+                    break;
+                case ("OnStopActivity", NpgsqlCommand command):
+                    activity.SetTag("succeeded", true);
+                    break;
+                case ("OnException", Exception exception):
+                    activity.SetTag("succeeded", false);
+                    activity.SetTag("stackTrace", exception.StackTrace);
```

Comment:
I know this follows the HTTP example, but note that we already have the `exception.stacktrace` tag, as per standard practice - so this may not be the best example.

Reply:
It would be redundant information on the trace as a whole, but given this is setting an attribute/tag on the span/`Activity` rather than on the exception `ActivityEvent`, I think it could be a valid thing to do.